Simulator/tft_simulator.py

```diff
@@ -234,6 +235,11 @@ def step(self, action):
             return
 
         agent = self.agent_selection
+
+        # if player is afk then pass action
+        if self.player_manager.player_states[agent].is_afk:
```

Comment:
I don't like this being here. This is on every action and will slow down the simulator. Use the implementation we were talking about before with truncated agents. That allows for a very fast and expandable implementation of this augment.